### DISCUSSION TITLE
How to show ToolTip for disabled Control

### DISCUSSION THREAD

**Discussion by ShrutiJaiswal1494** [AvaloniaUI/Avalonia#5790] [Q&A] [answered]
I have a button which is enabled/disabled based on a textbox. I want to show the ToolTip even when the button is disabled. In WPF there is an option `ToolTipService.ShowOnDisabled = True` to do this but it doesn't work for Avalonia.

```
<Border ToolTip.Placement="Bottom" >
  <ToolTip.Tip>
    <StackPanel>
      <TextBlock FontWeight="DemiBold">Info</TextBlock>
      <TextBlock>Info text here.</TextBlock>
    </StackPanel>
  </ToolTip.Tip>
  <Button Name="filterBtn" IsEnabled="{Binding ElementName=filterTextBox, Path=Text.Length}" >
    <Image Source="/Assets/Filter/filter.png" Height="20"/>
  </Button>
</Border>
```

**Comment by maxkatz6** [comment]
https://github.com/AvaloniaUI/Avalonia/issues/3847

**Comment by nref** [comment, marked as answer]
There is now https://api-docs.avaloniaui.net/docs/M_Avalonia_Controls_ToolTip_SetShowOnDisabled
